src/locales/en/translations.json

```
@@ -85,14 +85,14 @@
     "info-link": "<0>Use our contact-form or write an e-mail to <1>beep@satellytes.com</1></0>",
     "info": "Use our contact-form or write an e-mail to beep@satellytes.com",
     "name": "Name",
-    "email": "E-mail-adresse",
+    "email": "E-mail-adress",
```

`Adresse` is `address` in English (double d, double s). Also I'm not sure about the second hyphen (`-`). Personally I'd remove it. Maybe you can clarify with Eric?
```suggestion
    "email": "E-mail address",
```